grammars/rust.cson

```diff
@@ -327,6 +327,21 @@
       '1': { 'name': 'entity.name.function.rust' }
     }
   }
+  {
+    'comment': 'Function call with type arguments'
+    'begin': '\\b([a-zA-Z_][a-zA-Z0-9_]*)\s*::\s*<'
```

Comment:
Could you re-use `    'comment': 'Function call'` instead of creating a new one?
Could you look at `Function definition` to figure out how to use `type_params`?